src/DistributedArrays.jl

```diff
@@ -861,8 +861,8 @@ end
 (-)(x::Number, A::DArray) = x .- A
 
 map_localparts(f::Callable, d::DArray) = DArray(i->f(localpart(d)), d)
-map_localparts(f::Callable, d1::DArray, d2::DArray) = DArray(d1) do I
-    f(localpart(d1), localpart(d2))
+map_localparts(f::Callable, d::DArray, ds::DArray...) = DArray(d) do I
+    f(localpart(d), map(localpart, ds)...)
```

Comment:
Isn't this covered by https://github.com/JuliaParallel/DistributedArrays.jl/blob/4e1a2804fcae0a4bde594f298cc3db484f894eca/src/DistributedArrays.jl#L965 ?